test_util/main.tf

```diff
@@ -0,0 +1,140 @@
+provider "aws" {}
+
+#Linux Installer path - place url with "pull/PR#" or "master" suffix here:
+variable "custom_dcos_download_path" {
+  type = "string"
+  default = "https://downloads.mesosphere.com/dcos-enterprise/testing/master/dcos_generate_config.ee.sh"
+}
+
+#Windows Installer path - place url with "pull/PR#" or "master" suffix here:
+variable "custom_dcos_download_path_win" {
+  type = "string"
+  default = "https://downloads.mesosphere.com/dcos-enterprise/testing/master/windows/dcos_generate_config_win.ee.sh"
+}
+
+variable "variant" {
+  type = "string"
+  default = "ee"
+}
+
+variable "windowsagent_num" {
+  type = "string"
+  default = "0"
+  description = "Defines the number of Windows agents for the cluster."
+}
+
+# Used to determine your public IP for forwarding rules
+data "http" "whatismyip" {
+  url = "http://whatismyip.akamai.com/"
+}
+
+locals {
+  cluster_name = "generic-dcos-ee-demo"
+}
+
+module "dcos" {
+  source  = "dcos-terraform/dcos/aws"
+  version = "~> 0.2.0"
+
+  providers = {
+    aws = "aws"
+  }
+
+  cluster_name        = "${local.cluster_name}"
+  ssh_public_key_file = "~/.ssh/id_rsa.pub"
+  admin_ips           = ["${data.http.whatismyip.body}/32"]
+
+  num_masters        = "1"
+  num_private_agents = "1"
+  num_public_agents  = "1"
+
+  dcos_instance_os        = "centos_7.5"
+  bootstrap_instance_type = "m4.xlarge"
+
+  dcos_variant              = "${var.variant}"
+  dcos_version              = "2.1.0-beta1"
+  dcos_license_key_contents = "${file("~/license.txt")}"
```

Comment:
This should be a [variable](https://learn.hashicorp.com/terraform/getting-started/variables.html)

Reply:
but we can fix it later